src/group/GroupContainer.js

```diff
@@ -1,10 +1,10 @@
 /* @flow */
-import { getOwnEmail, getUsers, getPresence } from '../selectors';
+import { getOwnEmail, getOtherUsers, getPresence } from '../selectors';
```

Comment:
I've used the 'sans me' term before to indicate a similar concept. The name was `normalizeRecipientsSansMe`. This one would then go like `getUsersSansMe`.

Reply:
That looks better. I'll change it.